### ISSUE TITLE
wabweb container exit code 1: /opt/whatsapp/bin/web_setup.sh: line 368: WACORE_PORT_6250-6253_TCP: invalid variable name

### ISSUE THREAD

**Issue by ronna** [WhatsApp/WhatsApp-Business-API-Setup-Scripts#101]
Following the docker compose install and using docker files wabweb container is not starting, see log below

waweb-1   | Uploading certificate to db
waweb-1   | Downloading certificate from db
waweb-1   | Installed ca cert at /var/lib/whatsapp/ca.pem
waweb-1   | Installed server cert at /var/lib/whatsapp/www.cert
waweb-1   | Rewrote cert file at /var/lib/whatsapp/server.cert
wacore-1  | D 2024-11-04 13:02:44.625 UTC 44 servicemessagestore_p.h:1350] Creating template_rate_limiting_rolling_window_counter table req_id=none
wacore-1  | D 2024-11-04 13:02:44.625 UTC 44 datastore.cpp:1587] Creating table "template_rate_limiting_rolling_window_counter" req_id=none
wacore-1  | D 2024-11-04 13:02:44.625 UTC 44 datastore.cpp:1647] "CREATE TABLE IF NOT EXISTS template_rate_limiting_rolling_window_counter (  timestamp BIGINT NULL  , template_id  VARCHAR(512) CHARACTER SET latin1  NOT NULL  , pacing_session BIGINT NULL   )" req_id=none
waweb-1   | SSL Hardened
waweb-1   | SSL enabled
waweb-1   | Provisioned certificates successfully
waweb-1   | Cleaned up staging area
wacore-1  | D 2024-11-04 13:02:44.652 UTC 44 datastore.cpp:1497] Creating indexes for "template_rate_limiting_rolling_window_counter" req_id=none
**waweb-1   | /opt/whatsapp/bin/web_setup.sh: line 368: WACORE_PORT_6250-6253_TCP: invalid variable name**

**Comment by zerow06** [comment]
Same issue here using WA_API_VERSION=2.59.2 and latest version of docker-compose file with Docker Desktop 4.35.1 (173168)

**Comment by bmm-alc** [comment]
same here
ubuntu 20.04
docker: 24.0.7-0ubuntu2~20.04.1
compose: 2.24.6+ds1-0ubuntu1~20.04.1

**Comment by bmm-alc** [comment]
I also opened a ticket on meta developer site wishing that help fixing the issue https://developers.facebook.com/community/threads/1090523782334534/?post_id=1090523785667867

**Comment by RonnieRocket147** [comment]
I had the same problem, try removing the lines

    ```network_mode: bridge```

**Comment by bmm-alc** [comment]
> I had the same problem, try removing the lines
> 
> ```
> network_mode: bridge
> ```

Yes I think this is because the default network bridge does not provide name resolution as stated in https://docs.docker.com/engine/network/drivers/bridge/#differences-between-user-defined-bridges-and-the-default-bridge
But I don't like the idea of patching file provided in the repo, and I did not find a way to patch using an override file.
If you have suggestion I'll take it

**Comment by Patrickl-Commbox** [comment]
There is any update\solution on this issue?
I also removed the links , and the error dissapear , but the coreapp container comes up with 0/tcp in the ports, even there are port mapping of 6250-6253 in the template yml.
